website/source/docs/guides/connect-production.md

```diff
@@ -130,7 +102,7 @@ connect {
 }
 ```
 
-This config change requires a restart which you can perform one server at a time
+This configuration change requires a Consul service restart, which you can perform one server at a time
```

Comment:
```suggestion
This configuration change requires a Consul server restart, which you can perform one server at a time
```

Reply:
Don't love this but I found "Consul service restart" confusing as we don't refer to Consul as a service much.